From dac561de902b23584b7a9acbbccce4dfa4e1628c Mon Sep 17 00:00:00 2001
From: Grygorii Strashko
Date: Thu, 22 Jul 2021 20:17:32 +0300
Subject: HACK: net: ethernet: ti: cpsw_new: add module descs_pool_size
 parameter

Add module parameter to configure descs pool size to work the same way as
legacy cpsw:
  ti_cpsw_new.descs_pool_size=<num desc>

Signed-off-by: Grygorii Strashko <grygorii.strashko@ti.com>
Signed-off-by: Vignesh Raghavendra <vigneshr@ti.com>
---
 drivers/net/ethernet/ti/cpsw_new.c | 2 ++
 1 file changed, 2 insertions(+)

(limited to 'drivers/net/ethernet/ti')

diff --git a/drivers/net/ethernet/ti/cpsw_new.c b/drivers/net/ethernet/ti/cpsw_new.c
index e67b38be6b6d..1fdffcc387cc 100644
--- a/drivers/net/ethernet/ti/cpsw_new.c
+++ b/drivers/net/ethernet/ti/cpsw_new.c
@@ -46,6 +46,8 @@ static int debug_level;
 static int ale_ageout = CPSW_ALE_AGEOUT_DEFAULT;
 static int rx_packet_max = CPSW_MAX_PACKET_SIZE;
 static int descs_pool_size = CPSW_CPDMA_DESCS_POOL_SIZE_DEFAULT;
+module_param(descs_pool_size, int, 0444);
+MODULE_PARM_DESC(descs_pool_size, "Number of CPDMA CPPI descriptors in pool");
 
 struct cpsw_devlink {
 	struct cpsw_common *cpsw;
-- 
cgit v1.2.3

